aws-predictions-tensorflow/src/test/resources/word-tokens.txt

```diff
@@ -0,0 +1,10 @@
+<PAD> 0
```

Comment:
Hmm I guess I could, but this was how the asset in the official sample tensorflow lite app was formatted. It wouldn't be a huge deal, since we can internalize how we want them to be formatted, but I feel like it could create confusion to customers who are directly copying the assets from the official tensorflow sample.

Reply:
Oh yea, okay! I agree. Let's stick with the TensorFlow sample.